### PR TITLE
Package user-setup.0.8

### PR DESCRIPTION
**Pull request by kit-ty-kate**
### `user-setup.0.8`

---
* Homepage: https://github.com/OCamlPro/opam-user-setup
* Source repo: git+https://github.com/OCamlPro/opam-user-setup.git
* Bug tracker: https://github.com/OCamlPro/opam-user-setup/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

File: packages/user-setup/user-setup.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+license: "ISC"
+tags: "org:ocamlpro"
+homepage: "https://github.com/OCamlPro/opam-user-setup"
+bug-reports: "https://github.com/OCamlPro/opam-user-setup/issues"
+depends: [
+  "ocaml" {>= "3.12.1"}
+  "ocamlfind" {build}
+  "cmdliner"
```
Comment (mseri) [inline packages/user-setup/user-setup.0.8/opam:11]:
```suggestion
  "ocamlbuild"
  "cmdliner"
```
due to
```
#=== ERROR while compiling user-setup.0.8 =====================================#
# context              2.3.0~alpha~dev | linux/s390x | ocaml-base-compiler.5.2.0 | pinned(https://github.com/ocaml-opam/opam-user-setup/archive/refs/tags/0.8.tar.gz)
# path                 ~/.opam/5.2/.opam-switch/build/user-setup.0.8
# command              ~/.opam/opam-init/hooks/sandbox.sh build make
# exit-code            2
# env-file             ~/.opam/log/user-setup-7-242ee9.env
# output-file          ~/.opam/log/user-setup-7-242ee9.out
### output ###
# ocamlbuild -r \
#   -tag debug \
#   -tag bin_annot \
#   -I . -I ocamltop -I emacs -I vim -I sublime -I gedit \
#   -use-ocamlfind -pkg unix -pkg re -pkg re.pcre -pkg cmdliner \
#    \
#   ousMain.native
# make: ocamlbuild: No such file or directory
# make: *** [Makefile:19: _build/ousMain.native] Error 127
```

File: packages/user-setup/user-setup.0.8/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+license: "ISC"
+tags: "org:ocamlpro"
+homepage: "https://github.com/OCamlPro/opam-user-setup"
+bug-reports: "https://github.com/OCamlPro/opam-user-setup/issues"
+depends: [
+  "ocaml" {>= "3.12.1"}
+  "ocamlfind" {build}
+  "ocamlbuild"
+  "cmdliner"
+  "re"
```
Comment (mseri) [inline packages/user-setup/user-setup.0.8/opam:13]:
```suggestion
  "re" {>= "1.7.2"}
```
due to
```
#=== ERROR while compiling user-setup.0.8 =====================================#
# context              2.3.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.05.0 | pinned(https://github.com/ocaml-opam/opam-user-setup/archive/refs/tags/0.8.tar.gz)
# path                 ~/.opam/4.05/.opam-switch/build/user-setup.0.8
# command              ~/.opam/opam-init/hooks/sandbox.sh build make
# exit-code            2
# env-file             ~/.opam/log/user-setup-7-4b40e1.env
# output-file          ~/.opam/log/user-setup-7-4b40e1.out
### output ###
# ocamlbuild -r \
#   -tag debug \
#   -tag bin_annot \
#   -I . -I ocamltop -I emacs -I vim -I sublime -I gedit \
#   -use-ocamlfind -pkg unix -pkg re -pkg re.pcre -pkg cmdliner \
#    \
#   ousMain.native
# ocamlfind ocamldep -package cmdliner -package re.pcre -package re -package unix -modules ousMain.ml > ousMain.ml.depends
# + ocamlfind ocamldep -package cmdliner -package re.pcre -package re -package unix -modules ousMain.ml > ousMain.ml.depends
# ocamlfind: Package `re.pcre' not found
# Command exited with code 2.
# make: *** [Makefile:19: _build/ousMain.native] Error 10
```

File: packages/user-setup/user-setup.0.8/opam [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+]
+depopts: ["tuareg" "merlin" "ocp-indent" "ocp-index"]
+available: opam-version >= "2.1"
+flags: plugin
```
Comment (mseri) [inline packages/user-setup/user-setup.0.8/opam:17]:
```suggestion
conflicts: [ "ocaml-option-bytecode-only" ]
flags: plugin
```

File: packages/user-setup/user-setup.0.8/opam [hunk 4]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+license: "ISC"
+tags: "org:ocamlpro"
+homepage: "https://github.com/OCamlPro/opam-user-setup"
+bug-reports: "https://github.com/OCamlPro/opam-user-setup/issues"
```
Comment (avsm) [inline packages/user-setup/user-setup.0.8/opam:7]:
```suggestion
bug-reports: "https://github.com/ocaml-opam/opam-user-setup/issues"
```

**Pull request merged**